examples/TypeScriptMessaging/android/build.gradle

```diff
@@ -38,6 +38,7 @@ allprojects {
             }
         }
         google()
+        jcenter()
```

Comment:
See other comment about jcenter

Reply:
This is NOT blocking for the release: @khushal87 please investigate and remove/replace

Reply:
I don't remember how I introduced this but as far as I know removing this crashed the app on the build. I will investigate it further.

Reply:
Error without `jcenter()`:

```
> Task :react-native-video:generateDebugRFile FAILED

Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

See https://docs.gradle.org/7.2/userguide/command_line_interface.html#sec:command_line_warnings
192 actionable tasks: 2 executed, 190 up-to-date

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-video:generateDebugRFile'.
> Could not resolve all files for configuration ':react-native-video:debugCompileClasspath'.
   > Could not find com.google.android.exoplayer:exoplayer:2.13.2.
     Required by:
         project :react-native-video
   > Could not find com.google.android.exoplayer:extension-okhttp:2.13.2.
     Required by:
         project :react-native-video

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 32s

error Failed to install the app. Make sure you have the Android development environment set up: https://reactnative.dev/docs/environment-setup.
Error: Command failed: ./gradlew app:installDebug -PreactNativeDevServerPort=8081

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-video:generateDebugRFile'.
> Could not resolve all files for configuration ':react-native-video:debugCompileClasspath'.
   > Could not find com.google.android.exoplayer:exoplayer:2.13.2.
     Required by:
         project :react-native-video
   > Could not find com.google.android.exoplayer:extension-okhttp:2.13.2.
     Required by:
         project :react-native-video

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 32s

    at makeError (/Users/khushalagarwal/Projects/stream-chat-react-native/examples/SampleApp/node_modules/execa/index.js:174:9)
    at /Users/khushalagarwal/Projects/stream-chat-react-native/examples/SampleApp/node_modules/execa/index.js:278:16
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async runOnAllDevices (/Users/khushalagarwal/Projects/stream-chat-react-native/examples/SampleApp/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/runOnAllDevices.js:109:5)
    at async Command.handleAction (/Users/khushalagarwal/Projects/stream-chat-react-native/examples/SampleApp/node_modules/@react-native-community/cli/build/index.js:192:9)
```

Solution used: https://github.com/react-native-video/react-native-video/issues/2468#issuecomment-913709636